Harvest.Web/Controllers/PermissionsController.cs

```diff
@@ -15,13 +15,16 @@
 
 namespace Harvest.Web.Controllers
 {
+
     [Authorize(Policy = AccessCodes.FieldManagerAccess)]
     public class PermissionsController : SuperController
     {
         private readonly AppDbContext _dbContext;
         private readonly IIdentityService _identityService;
         private readonly IUserService _userService;
 
+        public string TeamSlug => ControllerContext.RouteData.Values["team"] as string;

```

Comment:
move to super controller since this will be used in multiple controllers eventually